lib/api/core.dart

```diff
@@ -10,6 +10,24 @@ import '../model/binding.dart';
 import '../model/localizations.dart';
 import 'exception.dart';
 
+/// The Zulip Server version below which we should refuse to connect.
+///
+/// When updating this, also update [kMinSupportedZulipFeatureLevel]
+/// and the README.
+const kMinSupportedZulipVersion = '4.0';
+
+/// The Zulip feature level reserved for the [minSupportedZulipVersion] release.
```

Comment:
nit: the only `minSupportedZulipVersion` I can find is the parameter of `errorServerVersionUnsupportedMessage`, but it is not a defined symbol within the dartdoc's context; I think this is supposed to be the "k"-prefixed constant?

Reply:
Indeed, thanks for the catch!